client/extensions/woocommerce/state/sites/settings/products/reducer.js

```diff
@@ -5,12 +5,14 @@
  */
 
 import { createReducer } from 'state/utils';
-import { LOADING } from 'woocommerce/state/constants';
+import { ERROR, LOADING } from 'woocommerce/state/constants';
 import { updateSettings } from '../helpers';
 import {
 	WOOCOMMERCE_SETTINGS_BATCH_REQUEST_SUCCESS,
 	WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST,
 	WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST_SUCCESS,
+	WOOCOMMERCE_SETTINGS_PRODUCTS_UPDATE_REQUEST_FAILURE,
```

Comment:
What about having a reducer for WOOCOMMERCE_SETTINGS_PRODUCTS_UPDATE_REQUEST that sets state to LOADING? Would that be helpful for things like Save button rendering/form blocking?

Reply:
Not sure if the UI will require that yet, so will add it in later if needed.